optaplanner-quickstarts/quarkus-facility-location/src/main/java/org/acme/facilitylocation/persistence/FacilityLocationProblemRepository.java

```diff
@@ -0,0 +1,21 @@
+package org.acme.facilitylocation.persistence;
+
+import java.util.Optional;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.acme.facilitylocation.domain.FacilityLocationProblem;
+
+@ApplicationScoped
+public class FacilityLocationProblemRepository {
+
+    private FacilityLocationProblem facilityLocationProblem;
+
+    public Optional<FacilityLocationProblem> solution() {
+        return Optional.ofNullable(facilityLocationProblem);
+    }
+
+    public void update(FacilityLocationProblem facilityLocationProblem) {
+        this.facilityLocationProblem = facilityLocationProblem;
+    }
+}
```

Comment:
Proposal: database persistence later or never. Not important for this example.

Reply:
I am ok with never.
Timetabling already shows with it.